### DISCUSSION TITLE
最新0916正式版发布

### DISCUSSION THREAD

**Discussion by metercai** [metercai/SimpleSDXL#75] [Announcements]
2024-09-16  最新内容
- 解锁Flux的lora和可图lora。
- 自动修图开关移到增强修图标签内。修图更多情况是针对已有图片，而新生成图片往往并不知道是否需要修。
- 添加提示面板开关，可点击触发批量通配符面板。
- 反推提示词和图片参数提取两个标签移入参数设置栏，避免被锁在图像输入里。
- 添加基于预置包的基础模型过滤功能，避免预置包配置和模型不匹配情况发生。
- 修复Comfyd引擎的跳过和中断逻辑bug，使其效果与Fooocus引擎保持一致。
- 优化预置包参数和预置包导航。Flux可适用所有Flux模型并自动适配硬件环境。Flux🚀优选Hyp8Q5KM模型，在低配环境下具有速度与质量的平衡，同时也支持Flux Lora。可图🚀参数优化提升出图质量。新增两个平铺预置包，适用无缝贴图场景。
- 升级comfyd到最新版。优化下载安装和启动流程，强制基础包检测，并提供模型包安装脚本。拆分基础包缩小到25G，并按模型种类进行独立分包。
下载安装详情请见项目wiki：《SimpleSDXL创意生图应用指南》https://acnmokx5gwds.feishu.cn/wiki/QK3LwOp2oiRRaTkFRhYcO4LonGe

未来，我们的所有功能更新、操作指导和效果评测等内容都会更新在这个wiki上。请大家收藏此链接。每发新程序包时，会将wiki的最新内容生成PDF随程序包发送。

**Comment by DavidDragonsage** [comment]
Unfortunately, you have effectively locked out English language users.

First I downloaded and ran SimpleSDXL2_win.exe.7z0916. The readme file, Readme-SimpleSDXL创意生图应用指南-v0.2.pdf, is in Chinese. I found out how to translate a PDF using Google translate but the translation was poor. I came to a graphic showing which installation script to use:
install_and_run_安装模型再启动.bat
run_SimpleSDXL_常规启动.bat
run_without_update_不更新启动.bat
But of course I cannot translate an image!

Then I tried the link, https://acnmokx5gwds.feishu.cn/wiki/QK3LwOp2oiRRaTkFRhYcO4LonGe
I was able to create a new personal account on Feishu using Google but then Feishu does not allow page translation to English. I was able to highlight sections of the document for translation but then I came to the same graphic I had seen before, with no possibility of translation.

This is becoming far too difficult, and very frustrating! 🙁

**Reply by metercai** [reply]
> Unfortunately, you have effectively locked out English language users.
> 
> First I downloaded and ran SimpleSDXL2_win.exe.7z0916. The readme file, Readme-SimpleSDXL创意生图应用指南-v0.2.pdf, is in Chinese. I found out how to translate a PDF using Google translate but the translation was poor. I came to a graphic showing which installation script to use: install_and_run_安装模型再启动.bat run_SimpleSDXL_常规启动.bat run_without_update_不更新启动.bat But of course I cannot translate an image!
> 
> Then I tried the link, https://acnmokx5gwds.feishu.cn/wiki/QK3LwOp2oiRRaTkFRhYcO4LonGe I was able to create a new personal account on Feishu using Google but then Feishu does not allow page translation to English. I was able to highlight sections of the document for translation but then I came to the same graphic I had seen before, with no possibility of translation.
> 
> This is becoming far too difficult, and very frustrating! 🙁
 
These two documents are written based on the habits of Chinese users and may not be suitable for you. Just use it normally.

**Reply by metercai** [reply]
> Unfortunately, you have effectively locked out English language users.
> 
> First I downloaded and ran SimpleSDXL2_win.exe.7z0916. The readme file, Readme-SimpleSDXL创意生图应用指南-v0.2.pdf, is in Chinese. I found out how to translate a PDF using Google translate but the translation was poor. I came to a graphic showing which installation script to use: install_and_run_安装模型再启动.bat run_SimpleSDXL_常规启动.bat run_without_update_不更新启动.bat But of course I cannot translate an image!
> 
> Then I tried the link, https://acnmokx5gwds.feishu.cn/wiki/QK3LwOp2oiRRaTkFRhYcO4LonGe I was able to create a new personal account on Feishu using Google but then Feishu does not allow page translation to English. I was able to highlight sections of the document for translation but then I came to the same graphic I had seen before, with no possibility of translation.
> 
> This is becoming far too difficult, and very frustrating! 🙁

the English version, translation by Feishu
https://acnmokx5gwds.feishu.cn/wiki/Oczvw2T4HiTGAGkxw3VcoDwQnle

**Reply by DavidDragonsage** [reply]
Thank you MeterCai, that translation is a lot better than what I got from Google translate!

**Comment by DavidDragonsage** [comment]
So I had a look at install_and_run_安装模型再启动.bat using a text editor and that seemed to be the way to begin. When I ran it there was a long pause after a "file not found" error. I think I pushed the spacebar and then installation continued.

SimpleSDXL came up in Chinese and the "Language of UI" switch did not work. I remembered you had told me elsewhere to add 
--language en
to run.bat and that worked.

**Comment by metercai** [comment]
Old users can upgrade by restarting without downloading SimpleSDXL2/win.exe. 7z0916. The new package is for new users to use. It has been optimized in the installation and startup process.

**Reply by DavidDragonsage** [reply]
Do you mean with the auto-update when you use run_SImpleSDXL.bat?  I was surprised but it did not try to auto-update. That is why I did the full install using SimpleSDXL2/win.exe. 7z0916. 

This is the run_SimpleSDXL batch file I am using:
@echo off
python_embeded\python.exe -s SimpleSDXL\entry_with_update.py --models-root ../../SimpleModels --output-path E:/stable-diffusion-webui/outputs --language en
echo All done.
pause

**Comment by DavidDragonsage** [comment]
So what method could I have used to update my previous installation rather than doing a new install?

Are you referring to run_SimpleSDXL_常规启动.bat in the \SimpleAI\SimpleSDXL2_win_0916 folder?

**Comment by metercai** [comment]
<img width="844" alt="截屏2024-09-18 19 11 04" src="https://github.com/user-attachments/assets/8fc48197-d341-460f-9693-f3309dfe2371">

**Reply by DavidDragonsage** [reply]
Excellent information - thank you very much! 🙂

**Comment by devin2025** [comment]
用了一段时间了，感觉很好用。谢谢分享！
另外sd3.5和flux的lora太少了，能不能开放1/2个，或者给使用者提供修改代码的方案？根据自己的需求调用。
